src/clr-angular/utils/drag-and-drop/interfaces/draggable-event.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export interface ClrDraggableEvent {
+    // TODO: this interface will be expanded and customized as the implementation progresses.
+    dragDataStore: any;
```

Comment:
You want to have a type parameter on the data type:
```typescript
export interface ClrDraggableEvent<T> {
  dragDataStore: T;
}
```
It'll allow proper type checking both for us and for people using the feature.

Reply:
I've removed this property for now. So instead, it has only a `type` property for now. We could decide on this property once we progress to its implementation.